lua/neogit/status.lua

```diff
@@ -688,8 +688,23 @@ local discard = a.sync(function()
   __NeogitStatusRefresh(true)
 end)
 
-local function create(kind)
+local function create(...)
+  local args = {...}
+  local kind = nil
+
+  for _, val in pairs(args) do
+    local kindpos = string.find(val, "kind=")
+    if kindpos then
+      kind = string.sub(val, 6)
+    end
+  end
```

Comment:
Sorry, but could you extract the parsing out of the create function and change the signature back to how it is right now? You can then change mapping to:

```lua
require'neogit.status'.create(require'neogit.util'.parse_command_args(<f-args>))
```
Obviously the name I chose for the function is just a placeholder and you can change it to whatever you think fits best.